tests/sequential_calls.rs

```diff
@@ -0,0 +1,20 @@
+use mockiato::mockable;
+
+#[mockable]
+trait Foo {
+    fn bar(&self) -> bool;
+}
+
+#[test]
+fn sequential_calls() {
+    let mut mock = FooMock::new();
+
+    mock.expect_bar().returns(true);
+    mock.expect_bar().returns(false);
```

Comment:
Also test, that other method's expectations are still unordered.

Reply:
87070c1